api/models/Site.js

```diff
@@ -5,7 +5,11 @@
 * @docs        :: http://sailsjs.org/#!documentation/models
 */
 
-var DEFAULT_BUCKET = 'federalist.18f.gov.s3-website-us-east-1.amazonaws.com';
+var env = process.env.NODE_ENV;
+
+var DEFAULT_BUCKET = env === 'staging' ?
+  'federalist-staging.18f.gov.s3-website-us-east-1.amazonaws.com' :
+  'federalist.18f.gov.s3-website-us-east-1.amazonaws.com';
```

Comment:
Do we want to make the bucket configuration come from the environment so we don't have this configuration/switch in code? If we always get the settings from the same environment variable, we don't need to know if we're dev/staging/production or whatever.

I'm fine keeping this for now as we had the production config here already; it's something to think about.